fcl/dunefdvd/gen/solar/prodmarley_flat_dunevd10kt_1x8x14_3view_30deg.fcl

```diff
@@ -0,0 +1,5 @@
+#include "prodmarley_nue_spectrum_dunevd10kt_1x8x14_3view_30deg.fcl"
+
+physics.producers.marley: @local::dune_marley_flat
+physics.producers.marley.marley_parameters.source.Emax: 30.
+physics.producers.marley.marley_parameters.source.E_bin_lefts: [2.]
```

Comment:
This gets set in a bunch of fcls repeatedly, and it isn't clear from the name of the fcl that these should be the parameters.  If these parameters are common to a bunch of fcls then they should be set once in an upstream fcl and all other fcls should inherit

Reply:
Done